appveyor.yml

```
@@ -1,8 +1,20 @@
-image: Visual Studio 2019
+image: 
+  - Visual Studio 2019
+  - Ubuntu
+
 configuration: Release
     
-build:
-  project: basisu.sln
-  
+test_script:
+  - ps: |
+      New-Item -Path . -Name "build" -ItemType "directory"
+      cd build
+      cmake --version
+      cmake ../ -DCMAKE_BUILD_TYPE:STRING="Release"
+      cmake --build . --config Release
+      cd ../
+
+build: off
```

See https://www.appveyor.com/docs/appveyor-yml/

```suggestion
build_script:
  - ps: |
      New-Item -Path . -Name "build" -ItemType "directory"
      cd build
      cmake --version
      cmake ../ -DCMAKE_BUILD_TYPE:STRING="Release"
      cmake --build . --config Release
      cd ../
```